README.md

```diff
@@ -137,3 +137,30 @@ Open your browser and navigate to [http://localhost:4200](http://localhost:4200)
 ---
 
 [Details regarding architecture](ARCHITECTURE.md)
+
+
+# The Open Mina documentation overview
+
+- [Why we are developing Open Mina](https://github.com/JanSlobodnik/pre-publishing/blob/main/why-openmina.md)
+- What is Open Mina?
+  - [Openmina Node](https://github.com/openmina/openmina/blob/main/README.md)
+  - [The Mina Web Node](https://github.com/openmina/webnode/blob/main/README.md)
+- Core components
+  - [P2P communication](https://github.com/openmina/openmina/blob/documentation/docs/p2p_service.md)
+    - [GossipSub](https://github.com/openmina/mina-wiki/blob/3ea9041e52fb2e606918f6c60bd3a32b8652f016/p2p/mina-gossip.md)
+  - [Block production](https://github.com/JanSlobodnik/pre-publishing/blob/main/block-production.md)
+  - [Scan state](https://github.com/JanSlobodnik/pre-publishing/blob/main/scan-state.md)
```

Comment:
some of these docs are now in this repository right? these links need to be updated (using relative paths like @adonagy suggested bellow)